privaterelay/settings.py

```diff
@@ -227,11 +227,17 @@
     "api.apps.ApiConfig",
 ]
 
+API_DOCS_ENABLED = config("API_DOCS_ENABLED", False, cast=bool)
```

Comment:
```suggestion
API_DOCS_ENABLED = config("API_DOCS_ENABLED", False, cast=bool) or DEBUG
```
This should ensure that if we are in `DEBUG` mode we'll always get the swagger doc and the endpoints.